Source/LibNative/module.modulemap

```diff
@@ -0,0 +1,6 @@
+
+module LibNative {
+    header "CommonRuntimeError.h"
+    export *
+}
+
```

Comment:
Why not put it in https://github.com/awslabs/aws-crt-swift/tree/main/aws-common-runtime/config?

Reply:
The `config` module is required to build before `aws-c-common` because it is used for build configuration. Meanwhile, the `error` file uses error struct from `aws-c-common`, which introduces a dependency on `aws-c-common`. If we add the error file into config module, it would be a recursive dependency which is not allowed by the swift package.